dali/test/example_test.cc

```diff
@@ -20,38 +20,57 @@ namespace testing {
 
 
 class ExampleOperatorTestCase : public DaliOperatorTest {
+ protected:
+  ExampleOperatorTestCase() :
+          tlcpu(std::unique_ptr<TensorList<CPUBackend>>(new TensorList<CPUBackend>())),
+          tlgpu(std::unique_ptr<TensorList<GPUBackend>>(new TensorList<GPUBackend>())) {
+    std::call_once(once_flag_, [&]() {
+        tlcpu->Resize({{1}});
+        auto ptrcpu = tlcpu->mutable_data<int>();
+        ptrcpu[0] = 666;
+    });
+
+    in_ = TensorListWrapper(tlcpu.get());
+    out_ = TensorListWrapper(tlgpu.get());
+  }
+
+
+ private:
   GraphDescr GenerateOperatorsGraph() const noexcept override {
     GraphDescr graph("ExampleOp");
     return graph;
   }
 
 
  protected:
-  TensorListWrapper in_;  // fill it somewhere
-  TensorListWrapper out_;  // fill it somewhere
+  std::unique_ptr<TensorList<CPUBackend>> tlcpu;
+  std::unique_ptr<TensorList<GPUBackend>> tlgpu;
+  TensorListWrapper in_;
+  TensorListWrapper out_;
+  std::once_flag once_flag_;
```

Comment:
```suggestion
   std::once_flag setuop_once_flag_;
```
To make it meaningful.